### DISCUSSION TITLE
Data format licence

### DISCUSSION THREAD

**Discussion by overflorian** [OvertureMaps/data#305] [Discussion]
Hi,
It seems that you licenced this data format under "Copyright © Overture Maps Foundation, 2023" as I can see on this page https://docs.overturemaps.org

Would it be an option to switch it to an open licence such as Creative Commons, preferably CC-BY-SA ?

**Comment by mprioleau** [comment]
Yes, copyright for now while we are evaluating the license choices for the schema. We expect to  make a decision shortly. We are considering both share alike licenses as well as more permissive licenses. - Marc

**Comment by hrabbey** [comment]
CC BY-SA 2.0 would aid reuse and coordination with OpenStreetMap definitions. Given that "wiki.openstreetmap" shows up 7 times currently in the repository, license compatibility would seem to be important.  Refer: https://wiki.openstreetmap.org/wiki/Wiki_content_license

That seems to maintain compatibility with GeoJSON's license, which seems to be licensed CC BY 3.0 and BSD.

**Comment by RobSoetewey-TomTom** [comment]
The Schema license will be CC-BY 4.0
The data itself will have a different license depending on the source.

**Reply by overflorian** [reply]
Thanks for this very satisfying reply @RobSoetewey-TomTom
